float64 velocityRatio
float64 leftVelocity
float64 rightVelocity
float64 x
float64 y
float64 theta
float64 curvature
float64 costOfseed
